### PR TITLE
null schema support

### PR DESCRIPTION
**Pull request by mksample**
adds support for a schema not being specified or being an empty string

### CODE REVIEW

File: transmogrifier.js [hunk 1]
```diff
@@ -610,6 +613,7 @@ async function transmogrify(manifest) {
     const schemaEntryDatas = [];
     for (schemaEntry of manifest) {
         const entryDatas = [];
+        schemaEntry.schema = schemaEntry.schema ? schemaEntry.schema : "";
```
Comment (sarahsemkow) [inline transmogrifier.js]: you can use schemaEntry.schema ?? "" like above

File: transmogrifier.dart [hunk 2]
```diff
@@ -156,6 +156,9 @@ Future<Function> getSinkFunction(String name) async {
 }
 
 Future<String> getSchema(String path) async {
+  if (path == "") {
```
Comment (sarahsemkow) [inline transmogrifier.dart:159]: should this check for null/undefined, etc, if so you can use: if (!path)

**Review by sarahsemkow — COMMENTED** [review]
just some small changes!

File: transmogrifier.js (hunk 1 above)
Comment (mksample) [reply transmogrifier.js]: didn't know you could do that in js too

File: transmogrifier.dart (hunk 2 above)
Comment (mksample) [reply transmogrifier.dart:159]: probably a good idea, will add

Comment (mksample) [reply transmogrifier.dart:159]: actually im not sure the above would work for Dart. Im still a little rocky on Dart so I think keeping it as is is probably the safest bet for now

**Review by FelixNg21 — APPROVED** [review]

**Review by sarahsemkow — APPROVED** [review]

**Review by b-hsieh — APPROVED** [review]

**Pull request merged**